lib/mimoracle/hf_datasets/mimic_iii/_processing.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import re
+import sqlite3
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+import typer
+from tqdm import tqdm
+
+ROOT = Path(__file__).parent.parent.parent
+
+
+def main(
+    min_docs_per_patient: int = 3,
+    n_patients: Optional[int] = 1000,
+    mimic_db_path: Path = ROOT / "data" / "mimic.db",
+) -> None:
+    """Preprocess the MIMIC-III dataset into train and test CSV files.
+
+    Only the discharge summaries are kept, and only for `n_patients` patients with more than
+    `min_docs_per_patient` documents. The discharge summaries are then split into sections.
+
+    The script is based on the `mimic.db` SQLite database, which must be downloaded beforehand.
```

Comment:
Can you provide an URL for that?

Reply:
I won't link it in the code since it's on Arkhn's private drive but you can fin it on there in the AI folder